codemodder/codemods/transformations/clean_imports.py

```diff
@@ -247,22 +371,22 @@ def _triage_imports(all_imports, src_path):
 
     config = Config(src_paths=(src_path,))
 
-    for imp in all_imports:
+    for stmt in all_imports:
+        imp = stmt.body[0]
         section = isort.place.module(_get_name(imp), config)
 
         if section == isort.sections.FUTURE:
-            future.append(imp)
+            future.append(stmt)
```

Comment:
This isn't from your change specifically, but the multiple `elif`s here make this code feel like it would be better handled by a different data type.